chapter0/example_0_4_gaussian_distribution/example_0_4_gaussian_distribution.pde

```diff
@@ -0,0 +1,16 @@
+// The Nature of Code
+// Daniel Shiffman
+// http://natureofcode.com
+
+void setup() {
+  size(640, 240);
+  background(255);
+}
+
+void draw() {
+  //{!1} A normal distribution with mean 320 and standard deviation 60
```

Comment:
```suggestion
  // A normal distribution with mean 320 and standard deviation 60
```

This is some leftover mark up for the book's layout that is not necessary in the code examples!